File: android-app/app/src/main/java/com/tahanot/entities/StopSignRouteDestination.java
package com.tahanot.entities;

import org.simpleframework.xml.*;

@Root
public class StopSignRouteDestination {
	
	@Attribute(required=false)
	public String Text1;
	
	@Attribute(required=false)
    public String Text2;
	
	@Attribute(required=false)
    public int StopId;
}

File: azure-website/GtfsService/Entities/StopSignRouteDestination.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace GtfsBackend.Entities
{
    public class StopSignRouteDestination
    {
        [XmlAttribute]
        public string Text1;

        [XmlAttribute]
        public string Text2;

        [XmlIgnore]
        public int StopId;
    }
}
